cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt

```diff
@@ -41,7 +41,8 @@ class Cordformation : Plugin<Project> {
             val releaseVersion = project.findRootProperty<String>("corda_release_version")
                     ?: throw IllegalStateException("Could not find a valid declaration of \"corda_release_version\"")
             val maybeJar = project.configuration("runtime").filter {
-                "$jarName-$releaseVersion.jar" in it.toString() || "$jarName-enterprise-$releaseVersion.jar" in it.toString()
+                it.toString().contains("$jarName-enterprise-$releaseVersion(-*)?.jar".toRegex()) ||
+                it.toString().contains("$jarName-$releaseVersion(-.*)?.jar".toRegex())
```

Comment:
This one should probebly end with `\\.jar\$` because a literal `.` needs to be escaped in a regexp.

Reply:
The regex's have been tested to work.

Reply:
A `.` matches _any_ single character (including a literal `.`) whereas `\\.` would match _only_ a literal `.`. So we have different definitions of "work" here.